src/main/java/duke/command/TodoCommand.java

```diff
@@ -0,0 +1,23 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.Ui;
+import duke.Storage;
+import duke.task.Task;
+public class TodoCommand extends Command {
+    private String title;
+
+    public TodoCommand(String title) {
+        this.title = title;
+    }
+    @Override
```

Comment:
```suggestion
    }
    
    @Override
```
Whitespace here too! :)